config.js

```diff
@@ -238,114 +238,130 @@ exports.updateFromArgv = function() {
+                    console.log(`Warning: /--${argumentToTest} auto/ has been renamed to /--${argumentToTest} automatic/\nRead latest up to date available gtp2ogs arguments in OPTIONS-LIST (devel branch) for details : https://github.com/online-go/gtp2ogs/blob/devel/docs/OPTIONS-LIST.md`); 
+                }
+                if (e === "null") {
+                    indicatorDepreciation = true;
+                    console.log(`Warning: /--${argumentToTest} null/ has been renamed to /--${argumentToTest} automatic/\nRead latest up to date available gtp2ogs arguments in OPTIONS-LIST (devel branch) for details : https://github.com/online-go/gtp2ogs/blob/devel/docs/OPTIONS-LIST.md`); 
+                }
+            }
         }
     }
 
-    console.log("\n"); /*after final warning, we skip a line to make it more pretty*/
-
-    // - warning : avoid infinite games
+    isArgumentArrayEqualToAuto(argv.komi, this.indicatorConsoleDepreciation);
+    isArgumentArrayEqualToAuto(argv.komiranked, this.indicatorConsoleDepreciation);
+    isArgumentArrayEqualToAuto(argv.komiunranked, this.indicatorConsoleDepreciation);
 
-    if (!argv.nopause && !argv.nopauseranked && !argv.nopauseunranked) {
-        console.log("Warning : No nopause setting detected, games are likely to last forever"); // TODO : when --maxpaustime and co gets implemented, replace with "are likely to last for a long time"
+    // final DEPRECIATIONS [OK!] check indicator : display result
+    if (this.indicatorConsoleDepreciation) {
+        console.log("result : [DEPRECIATIONS ERROR(S)]\n");
+    
+    } else {
+        console.log("result : [OK!]\n");
     }
-    console.log("\n"); /*after last warning, we skip a line to make it more pretty*/
 
-    // - warning : depreciated features if used
+    /*** console : WARNINGS, dont use 3 settings of the same family (general, ranked, unranked) at the same time ***/
+    console.log("-----GENERAL/RANKED/UNRANKED-----");
+    this.indicatorConsoleWarningGeneralRankedUnranked = false;
 
-    if (argv.botid) {
-        console.log("Warning: --botid alias is no longer supported. Use --username instead.");
-    }
-
-    if (argv.bot) {
-        console.log("Warning: --bot alias is no longer supported. Use --username instead.");
-    }
-
-    if (argv.id) {
-        console.log("Warning: --id alias is no longer supported. Use --username instead.");
-    }
-
-    if (argv.minrankedhandicap || argv.minunrankedhandicap || argv.maxrankedhandicap || argv.maxunrankedhandicap) {
-        console.log("Warning: --min/max*+/-ranked/unranked*handicap argument is no longer supported.\nUse --min/max*handicap*+/-ranked/unranked instead.");
-    }
-
-    if (argv.maxtotalgames) {
-        console.log("Warning: --maxtotalgames argument has been renamed to --maxconnectedgames. Use --maxconnectedgames instead.");
+    function generalArgumentIsUsedWithSpecificAtTheSameTime(generalArgument,rankedArgument,unrankedArgument, indicatorWarningsGeneralRankedUnranked) {
```

Comment:
I like this, but maybe it could be taken even further and work similarly as my `alternatives` suggestion above? So the helper function signature would be:

```
function checkConflictingArgs(generalArg, alternatives) {
    ...
}
```

So you could give it any number of arguments that conflict with the general argument.

Reply:
i see, so using an array then :1st_place_medal: